src/plugins/intel_cpu/src/nodes/kernels/jit_kernel_base.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <math.h>
+#include <dnnl_extension_utils.h>
+#include <cpu/x64/jit_generator.hpp>
+
+namespace ov {
+namespace intel_cpu {
+
+using namespace dnnl::impl;
+namespace x64 = dnnl::impl::cpu::x64;
+
+template<typename jit_params, typename jit_call_args>
+class jit_kernel_base : public x64::jit_generator {
+ public:
+    using kernel_func = void (*)(const jit_call_args *);
+
+    explicit jit_kernel_base(x64::cpu_isa_t max_cpu_isa, const jit_params &jqp)
```

Comment:
Due to this is a part of CPU plugin, let's keep camelCase style.

Reply:
Is there any specific requirements for code style ?

In another PR one of the guys suggested "snake" case https://github.com/openvinotoolkit/openvino/pull/12880#discussion_r962270486
Also snake case is consistent with `jit_emitters`

I think it is lack of code style in CPUPlugin. My suggestion is to use "snake" case for all classes and functions related to `jit` and for all other operations to use "camel" case

Reply:
The snake style is a legacy from mkldnn and we should not follow them. We develop another product. Especially in the base class it can change the minds for other derived kernels and keep whole plugin in one style.

Reply:
@dmitry-gorokhov Do you have some requirements to code style ? It would be nice to have some document that describe it

Reply:
Let's leave it as it is. Docs update for code style and appropriate PR are expected soon. It will be fixed there.